app/forms/support/edit_course_form.rb

```diff
@@ -0,0 +1,94 @@
+module Support
+  class EditCourseForm
+    include ActiveModel::Model
+    include ActiveModel::Validations
+
+    FIELDS = %i[
+      course_code name
+    ].freeze
+
+    attr_accessor(*FIELDS)
+    attr_accessor :day, :month, :year
```

Comment:
```suggestion
    attr_accessor :start_date_day, :start_date_month, :start_date_year
```

Reply:
So the problem with renaming these attrs is that it makes the model incompatible with:

```
      <%= f.govuk_date_field(:start_date,  legend: { text: "Start date" }, width: 10) %>
```
The `day`, `month` and `year` are methods which used when rendering the form after erros are hit.

With that in mind I don't think  we should rename these. Wdyt?